src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea(firstLeg * secondLeg / 2);
+    }
```

Comment:
Method name 'obtainTheArea' is a verb phrase, which is not recommended for method names. Consider renaming it to 'calculateArea' to better reflect its purpose. Additionally, this method should not be public if it's only used internally within the class.